example/takeoff_land/CMakeLists.txt

```diff
@@ -9,18 +9,19 @@ set(CMAKE_CXX_FLAGS_DEBUG "-DDEBUG")
 
 # This finds thread libs on Linux, Mac, and Windows.
 find_package(Threads REQUIRED)
+message(${CMAKE_SOURCE_DIR})
 
 include_directories(
     # Not needed if installed system-wide
-    ../install/include
+    ${CMAKE_SOURCE_DIR}/../../install/include
```

Comment:
Thanks for doing the rename. However, I'm puzzled how you can start from the root source dir, then go back twice and find `install/include` there? What am I missing?

Reply:
Did you mean links to example in https://github.com/dronecore/DroneCore/blob/master/docs/build.md#build-example ?
Also, link to CMakeLists.txt in the last line is invalid. We shall update both of these links to either of the example: `fly_mission` or `takeoff_land`

Reply:
I can edit [build.md](https://github.com/dronecore/DroneCore/blob/master/docs/build.md) only after this pull requests get merged. Isn't it ?

Reply:
No, just add the doc change here please.